### ISSUE TITLE
[BUG] Empty multiple & sortable select fields have a blank item in them. 

### ISSUE THREAD

**Issue by graham73may** [bolt/bolt#6094]
Empty multiple & sortable select fields have a blank item in them when the field should be blank.

![image](https://cloud.githubusercontent.com/assets/6642498/20643856/716bb880-b41b-11e6-874b-24ebb746886d.png)

Deleting the blank one, adding in some real content and saving works fine. 

Details
-------

 - Relevant Bolt Version: 3.2.2
 - Install type: Composer install 
 - PHP version: 7.0
 - Used web server: Apache 
 - For UX/UI issues: Chrome latest stable

Reproduction
------------
```
 speakers:
            group: "Meta"
            label: "Speakers"
            multiple: true
            sortable: true
            type: select
            values: people/title
```

**Comment by graham73may** [comment]
#6096 did fix this issue for me though! :) 👍

**Comment by GawainLynch** [comment]
Fixed in #6096
